Packages/CommonUI/CommonUI/Classes/View/TextField/CustomSecureTextField.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TextFieldStyle.swift
+//  
+//
+//  Created by NamNH on 03/03/2022.
+//
+
+import SwiftUI
+
+public struct CustomSecureTextField: UIViewRepresentable {
+
+	let placeHolder: String
+	@Binding var text: String
+	@Binding var isRevealed: Bool
+	let isFocused: (Bool) -> Void
+	let onSubmit: () -> Void
+	var textChange: ((String) -> Void)? = nil
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/CommonUI/CommonUI/Classes/View/TextField/CustomSecureTextField.swift#L17"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>CustomSecureTextField.swift:17</code></td>
    </tr>
  </tbody>
</table>